src/providers/test-result/test-result.ts

```diff
@@ -11,13 +11,9 @@ import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
 @Injectable()
 export class TestResultProvider {
 
-  constructor(
-    private faultCountProvider: FaultCountProvider,
-  ) {}
-
-  calculateTestResult = (category: string,
-                         testData: CatBUniqueTypes.TestData| CatBEUniqueTypes.TestData): Observable<ActivityCode> => {
+  constructor(private faultCountProvider: FaultCountProvider) {}
 
+  public calculateTestResult(category: string, testData: object): Observable<ActivityCode> {
```

Comment:
This does nothing if a category other than B or B+E is passed.  In some other cases i've seen it default to processing as cat b...  not sure if it should return a fail for anything it doesn't recognise??

Reply:
probably should `throw` something

Reply:
done